src/processes.c

```diff
@@ -2083,20 +2086,27 @@ static int ps_read (void)
 			case 'T': stopped++;  break;
 			case 'W': paging++;   break;
 		}
+		if (ps_get_cmdline (
+		     pid, NULL, cmdline, sizeof(cmdline)) != NULL) {
+			cmdline_processes++;
+		}
 
 		ps_list_add (ps.name,
-				ps_get_cmdline (pid, ps.name, cmdline, sizeof (cmdline)),
+				ps_get_cmdline (
+				   pid, ps.name, cmdline, sizeof(cmdline)),
```

Comment:
Do you want to keep the original formatting here as well?
FWIW, since the original code has a long line, you can remove the break after the parenthesis in line 2089 as well.

Reply:
Done.

Reply:
Not quite the original formatting -- there are still diffs (should be `sizeof (cmdline)`, not `sizeof(cmdline)`).

Reply:
PTAL